docs/howto/operate/new-cluster/index.md

```diff
@@ -20,6 +23,10 @@ In that case, follow the instructions at [](new-hub).
 
 The following sections cover how to deploy a Kubernetes cluster on the cloud providers that we support.
 
+:::{note}
+Currently, only Google Cloud has automatic cluster deployments via Terraform, but we should append to the sections below as we implement this for other providers.
```

Comment:
> automatic cluster deployments via Terraform

There is a subtle thing here... we use terraform to create GCP clusters. And terraform is somehow "automatically" creating entities accordingly to some provided configuration, BUT, we are not "automatically" deploying terraform-based cluster vis a CI tool (such as a GH action).

Reply:
hmm good point - maybe instead of having this special case here, I should just add another section for Azure and AWS and we can explicitly say how to deploy clusters there?

Reply:
@damianavila what do you think about the latest updates that explain our process for each of the major providers? (note that the Azure one is pretty much blank, I couldn't find information about how to deploy there, so I've decided to just leave a pointer to Yuvi's PR)

Reply:
I think that is OK, I left just one additional, and hopefully, clarifying comment about eksctl.